frontend/src/views/main/project/modals/ConfirmModal.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
For this file, we have a utility already, you can look in AdminUsers.vue line 137 for an example

Reply:
I'm not sure how I can get the `$dialog` reference with the vue 3 syntaxe

Reply:
Yeah it's true that `vuetify-dialog` doesn't support composition API. I started looking at `vue-final-modal` to replace it, but didn't finish the investigation. Kevin, could you maybe have a look too?
https://v2.vue-final-modal.org/dynamic-modal/?utm_source=pocket_reader